docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+version: '2'
```

Comment:
é a versão do cocker.composer.yml, [veja mais aqui](https://medium.com/@giorgioto/docker-compose-yml-from-v1-to-v2-3c0f8bb7a48e#.5t8zm9tvq).